### PR TITLE
Assessment metadata API- fix category to categories

### PR DESCRIPTION
**Pull request by Eitan-Lev**
update existing version to fix swagger quality issue- incorrect property name

### CODE REVIEW

**Comment by allenjzhang** [comment]
Please review and address model validation errors. One more thing is please do not delete the PR template. You should read thru and check the appropriate boxes. bot leverages those choices to apply appropriate labels. Thanks

File: specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AssessmentsMetadata/CreateAssessmentsMetadata_subscription_example.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/26a828e1-e88f-464e-bbb3-c134a282b9de",
```
Comment (raych1) [inline specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AssessmentsMetadata/CreateAssessmentsMetadata_subscription_example.json:9]: Regard to `model validation` errors,  this property is readonly and must be removed from request.

File: specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/examples/AssessmentsMetadata/CreateAssessmentsMetadata_subscription_example.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
         "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/26a828e1-e88f-464e-bbb3-c134a282b9de",
```
Comment (raych1) [inline specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/examples/AssessmentsMetadata/CreateAssessmentsMetadata_subscription_example.json:9]: Same here for the property to be removed in request.

**Review by JeffreyRichter — APPROVED** [review]

**Review by allenjzhang — DISMISSED** [review]
Please review the linter warning and add appropriate title/description. The example should also be updated as Ray indicated to fix the model validation. Thanks

**Review by allenjzhang — CHANGES_REQUESTED** [review]
Please fix model validation errors.

**Review by allenjzhang — APPROVED** [review]

**Comment by Eitan-Lev** [comment]
my colleague will take this PR, he will comment here after our weekend

**Pull request closed without merging**